content/cli/orch_cli/deployments.md

```diff
@@ -500,6 +500,192 @@ Delete labels from a specific deployment.
+                           Supports any timezone in the [tz database](en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+
+Valid relative time expressions are of the form `+<integer> minutes|hours|days|weeks|months|years` or a concatenation of several such, e.g. `+1 year +3 months`. 
+These can be also written without a space after the number, without the final `s`, or using the short forms `min|h|d|w|mo|y`. 
+
+
+#### schedule get
+
+##### Usage
+
+`cfy deployments schedule get [OPTIONS] NAME`
+
+Retrieve information for a specific deployment schedule.
+
+`NAME` is the name of the deployment schedule.
+
+##### Optional flags
+All the regular optional flags available for `list` commands, in addition to:
+
+*  `--preview INTEGER`   -  Preview the N next dates for the workflow execution to run.
+
+
+#### schedule create
+
+##### Usage
+
+`cfy deployments schedule create [OPTIONS] WORKFLOW_ID`
+
+Schedule the execution of a workflow on a given deployment.
+
+`WORKFLOW_ID` is the name of the workflow.
+
+##### Mandatory flags
+
+*  `-d, --deployment-id TEXT`   -  The unique identifier for the deployment.
+*  `-s, --since TEXT`           -  The earliest possible time to run. Supported formats:
+                                   `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+
+##### Optional flags
+
+*  `-n, --schedule-name TEXT`   -  A name for the schedule. If not provided, defaults to `{deployment-id}_{workflow-id}`.
+*  `-p, --parameters TEXT`      -  Parameters for the workflow 
+                                   (Can be provided as wildcard based paths (*.yaml, /my_inputs/ etc.) to YAML files, a JSON string or as `key1=value1;key2=value2`). This argument can be used multiple times.
+*  `--allow-custom-parameters`  -  Allow passing custom parameters (which were not defined in the workflow's schema in the blueprint) to the execution.
+*  `-f, --force`                -  Execute the workflow even if there is an ongoing execution for the given deployment.
+*  `--dry-run`                  -  If set, no actual operations will be performed. This only prints the executed tasks, without side effects.
+*  `--wait-after-fail INTEGER`  -  When a task fails, wait this many seconds for already-running tasks to return [default: 600].
+*  `-u, --until TEXT`           -  The latest possible time to run. Supported formats: 
+                                   `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+*  `--tz TEXT`                  -  The timezone to be used for scheduling, e.g. `EST` or `Asia/Jerusalem`. By default, the local timezone will be used. 
+                                   Supports any timezone in the [tz database](en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+*  `-r, --recurrence TEXT`      -  Recurrence on the scheduled execution. e.g. `2 weeks`, `30 min` or `1d`. You cannot use this argument with arguments: [rrule]
```

Comment:
"with arguments: [rrule]" -> "with the argument `rrule`".

Holds for other places too.

Reply:
This is consistent with the convention we have throughout the CLI docs